### PR TITLE
Fix the input check in groupby.

### PR DESCRIPTION
**Pull request by ueshin**
Fixes the input check in `groupby`.

### CODE REVIEW

File: databricks/koalas/tests/test_groupby.py [hunk 1]
```diff
@@ -139,53 +139,84 @@ def test_groupby_simple(self):
...
         )
         self.assert_eq(
-            kdf.groupby(("x", "a"))[[("y", "c")]].sum().sort_index(),
-            pdf.groupby(("x", "a"))[[("y", "c")]].sum().sort_index(),
+            kdf.groupby((10, "a"))[[(20, "c")]].sum().sort_index(),
+            pdf.groupby((10, "a"))[[(20, "c")]].sum().sort_index(),
+        )
+
+        # TODO: a pandas bug?
+        #  expected = pdf.groupby((10, "a"))[(20, "c")].sum().sort_index()
```
Comment (itholic) [inline databricks/koalas/tests/test_groupby.py:203]: nit of nit: one more space here in the comment 😅

File: databricks/koalas/tests/test_groupby.py [hunk 2]
```diff
@@ -2345,10 +2399,6 @@ def test_get_group(self):
         self.assertRaises(
             KeyError, lambda: kdf.groupby(("B", "class"))[("A", "name")].get_group("fish")
         )
-        self.assertRaises(
-            KeyError,
-            lambda: kdf.groupby(("B", "class"))[("A", "name")].get_group(["bird", "mammal"]),
-        )
```
Comment (itholic) [inline databricks/koalas/tests/test_groupby.py:2351]: You remove this because this is duplicated test with below ?

**Comment by itholic** [comment]
LGTM. Anyway I think it's good to merge regardless of two nit comments.

**Review by itholic — APPROVED** [review]

File: databricks/koalas/tests/test_groupby.py (hunk 1 above)
Comment (itholic) [reply databricks/koalas/tests/test_groupby.py:203]: Yes, and I think It's pandas bug. I'll report to their repository

Comment (itholic) [reply databricks/koalas/tests/test_groupby.py:203]: Anyway, `pdf.groupby((10, "a"))[[(20, "c")]].sum()` seems working.

```python
>>> pdf
  10      20 30
   a  b    c  d
0  1  4  4.0  a
1  2  2  2.0  b
3  6  7  7.0  c
5  4  3  3.0  d
6  4  3  NaN  e
8  6  1  1.0  f
9  4  1  1.0  g
9  3  1  1.0  h
9  7  2  2.0  t

>>> pdf.groupby((10, "a"))[[(20, "c")]].sum()
          20
           c
(10, a)
1        4.0
2        2.0
3        1.0
4        4.0
6        8.0
7        2.0
```

Comment (itholic) [reply databricks/koalas/tests/test_groupby.py:203]: Oh, Sorry. nvm above. It's different from `pdf.groupby((10, "a"))[(20, "c")].sum()` anyway.

**Review by HyukjinKwon — APPROVED** [review]

**Pull request merged**